src/components/ebay-progress-bar-expressive/progress-bar-expressive.stories.ts

```diff
@@ -0,0 +1,162 @@
+                },
+            },
+        },
+        messages: {
+            control: {
+                type: "array",
+            },
+            description: `Short messages to display above the progress bar. Specify the text to display and, optionally, a custom duration. By default, messages display for ${messageDurationStandard}ms. When the user prefers reduced motion, the message will display for ${messageDurationReducedMotionMultiplier} times its duration.`,
+            table: {
+                defaultValue: {
+                    summary: "[]",
+                },
+            },
+        },
+        size: {
+            type: "enum",
+            control: {
+                type: "select",
+            },
+            options: ["large", "medium"],
+            description: "Message text size",
+            table: {
+                defaultValue: {
+                    summary: "large",
+                },
+            },
+        },
+    },
+};
+
+export const Default = buildExtensionTemplate(
+    DefaultTemplate,
+    DefaultTemplateCode,
+);
+
+export const WithSingleMessage = Template.bind({});
+WithSingleMessage.args = {
+    messages: [{ text: "We're processing your order" }],
+};
+WithSingleMessage.parameters = {
+    docs: {
+        source: {
+            code: tagToString(
+                "ebay-progress-bar-expressive",
+                WithSingleMessage.args,
+            ),
+        },
+    },
+};
+
+export const WithMessages = buildExtensionTemplate(
+    MessagesTemplate,
+    MessagesTemplateCode,
+);
+WithMessages.args = {
+    messages: exampleMessages,
+};
+
+export const WithLongMessage = Template.bind({});
+WithLongMessage.args = {
+    messages: [
+        { text: "Messages should be one line..." },
+        { text: "Sometimes that's hard to guarantee, though.", duration: 2500 },
+        { text: "That's okay!" },
+    ],
+};
+WithLongMessage.parameters = {
+    docs: {
+        source: {
+            code: tagToString(
+                "ebay-progress-bar-expressive",
+                WithLongMessage.args,
+            ),
+        },
+    },
+};
+
+export const WithCustomTiming = buildExtensionTemplate(
+    CustomTimingTemplate,
+    CustomTimingTemplateCode,
+);
+WithCustomTiming.args = {
+    messages: [
+        { text: "Display for 2 seconds", duration: 2000 },
+        { text: "Display for 3 seconds", duration: 3000 },
+        { text: "Display for 4 seconds", duration: 4000 },
+    ],
+};
+
+export const Localized = buildExtensionTemplate(
+    LocalizedTemplate,
+    LocalizedTemplateCode,
+);
+Localized.args = {
+    a11yText: "Cargando...",
+    messages: [
+        { text: "Espera..." },
+        { text: "Estamos procesando tu pedido", duration: 2000 },
+        { text: "Sólo un momento más" },
+    ],
+};
+
+export const MediumSize = buildExtensionTemplate(
+    MediumTextTemplate,
+    MediumTextTemplateCode,
+);
+MediumSize.args = {
+    size: "medium",
+    messages: exampleMessages,
+};
```

Comment:
I don't think we need this example. We have toggles for toggling medium/large. As long as we spread the input to the other stories we don't need to have this.

Reply:
Size doesn't just control the text size but also whether the message immediately appears - medium text messages appear immediately, while large text messages animates in after a short delay. Was thinking it would be useful to have a story to highlight this behavior. To your point, though, this behavior can also be seen when toggling the size in the Storybook controls. Just wondering if that might get missed unless people go digging for it.